crates/base-db/Cargo.toml

```diff
@@ -1,7 +1,8 @@
 [package]
 name = "base-db"
 version = "0.0.0"
-description = "TBD"
+repository = "https://github.com/rust-lang/rust-analyzer"
+description = "Basic database traits. The concrete DB is defined by `ra_ap_ide`."
```

Comment:
Bah, feels like this should say `ide`.

Reply:
I changed it to `ra_ap_ide` as that's the name that the crate is currently published as. As such anyone reading the description on crates.io would be tempted to search for `ide` but not find anything.

Happy to change it back to `ide` though, if that's the consensus of what it should be referred to as.

Reply:
Can we just do something like `ide/ra_ap_ide` then?